packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -72,23 +85,33 @@ class Trade extends React.Component {
         });
     };
 
-    onTrySyntheticIndicesClick = () => {
-        this.setState(
-            {
-                try_synthetic_indices: true,
-            },
-            () => {
+    onTryOtherMarkets = async () => {
+        if (this.props.is_eu) {
+            this.setState({ try_open_markets: true }, () => {
+                this.setState({
+                    try_open_markets: false,
+                });
+            });
```

Comment:
(question) I know this might be out of the scope for this PR, but what are we trying to achieve here?

Reply:
@shayan-binary-2 would be a better candidate to answer this but yea we want to open markets side bar when a user clicks on `Try other markets`.  Just like `synthetic_indices` as before.

Reply:
@reza-fs to make `market dropdown` open, we should pass `open` prop to it, but if this prop keep `true`, the `market selector` keep opening on any prop change. that's why we should make it true and then false it afterward.

Reply:
Shouldn't this be handled inside the dropdown so other prop change won't affect re-opening things?

Reply:
@shayan-binary-2 :point_up_2:

Reply:
@reza-fs as we talked about, it's an improvement. could we ignore it for now?

Reply:
OK then I mark this as resolved.